doc/corewar-glossary.html

```
@@ -0,0 +1,675 @@
+<hr>
+
+<dt><a name="DClear">D-Clear</a></dt>
+
+<dd>A <a href="#C">1c</a> <a href="#CoreClear">core-clear</a> which alternates between
+<code class="code88">DAT</code> bombs and decrementing core.
+
+<pre class="code94">
+    <span class="label">example</span>   ...
+
+              spl   0
+    <span class="label">clear</span>     mov   bomb,  >example
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)